src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int num;
+
+    public Ball(String color,int num) {
+        this.color = color;
+        this.num = num;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public int getNum() {
+        return num;
+    }
+
+    @Override
+    public String toString() {
+        return "Color: " + getColor() + " \n" + "Number: " + getNum() + "\n";

```

Comment:
you have direct access to variables inside the class. 
```suggestion
        return "Color: " + color + " \n" + "Number: " + number + "\n";

```